resources/assets/wailaharvestability/lang/en_US.lang

```diff
@@ -30,3 +30,9 @@ option.harvestability.silktouchability=Silk Touchability
 option.harvestability.silktouchability.sneakingonly=Sneak only: Silk Touchability
 option.harvestability.harvestlevelnum=Harvest level no.
 option.harvestability.harvestlevelnum.sneakingonly=Sneak only: Harvest level no.
+
+# harvestlevels
+wailaharvestability.harvestlevel0=Wood
+wailaharvestability.harvestlevel1=Stone
+wailaharvestability.harvestlevel2=Iron
+wailaharvestability.harvestlevel3=Diamond
```

Comment:
Thats for this right?
https://github.com/MakerTim/WailaHarvestability/blob/f6d9df5e7d8a45f45a228fa7a8c3db1e3b9f1004/java/squeek/wailaharvestability/helpers/StringHelper.java#L39